internal/config/config.go

```diff
@@ -14,6 +14,7 @@ type Config struct {
 	Db      DbConfig          `mapstructure:"db"`
 	Queue   queue.QueueConfig `mapstructure:"queue"`
 	Metrics MetricsConfig     `mapstructure:"metrics"`
+	External ExternalConfig    `mapstructure:"external"`
```

Comment:
```suggestion
	Metrics MetricsConfig     `mapstructure:"metrics"`
	OrdinalsService OrdinalsServiceConfig   `mapstructure:"ordinals"`
```

Let's be explicit here, "external" is not a descriptive name.